azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/models/MessageInfoModel.kt

```diff
@@ -11,15 +11,15 @@ import org.threeten.bp.OffsetDateTime
 internal data class MessageInfoModel(
     val id: String?,
     val internalId: String? = null,
-    val messageType: ChatMessageType,
+    val messageType: ChatMessageType?,
```

Comment:
Why would this be null? Can we assign a non-null type in these cases (e.g. unknown)

Reply:
when we receive event notification from native SDK fro edited and deleted event, message type is null. Creating unknown will be a new type. I think we can follow native SDK.